core/modules/parser/QSMySqlListener.cc

```diff
@@ -1999,6 +1999,10 @@ class SelectExpressionElementAdapter :
         _valueExpr = valueExpr;
     }
 
+    virtual void handleUid(string const & string) {
```

Comment:
I guess the name of the parameter should be more descriptive, something like `uid`. That `string` is too generic. Also, remove extra space in `const &`.

Reply:
And the second note or question - is this method overriding the one from the base class/interface `handleUid`? If so then it should be declared as:
```
void handleUid(string const& uid) override {
```